master/internal/hpimportance/manager.go

```diff
@@ -66,16 +72,31 @@ type manager struct {
 }
 
 // NewManager initializes the master actor (of which there should only be one instance running).
-func NewManager(db *db.PgDB, system *actor.System, config HPImportanceConfig) actor.Actor {
+func NewManager(db *db.PgDB, system *actor.System, config HPImportanceConfig, masterRoot string,
+) (actor.Actor, error) {
+	// growforest should either be installed in PATH (when running from source) or package with the
+	// master (when running from binary packages).
+	growforest := path.Join(masterRoot, "growforest")
+	_, err := os.Stat(growforest)
+	if os.IsNotExist(err) {
+		growforest = "growforest"
+	}
+
+	workingDir := "/tmp/determined/growforest"
```

Comment:
I'd put this and all the other file related constants in a 
```
const (
    	workingDir := "/tmp/determined/growforest"
)
```
block together with really good comments.